conf/application.conf

```diff
@@ -132,6 +132,8 @@ mongodb {
 urls {
   login         = "http://localhost:9949/auth-login-stub/gg-sign-in"
   loginContinue = "http://localhost:9485/manage-transit-movements"
+  logout  = "http://localhost:9025/gg/sign-out"
+  enrolmentKey  = "HMCE-NCTS-ORG"
```

Comment:
This is held within Urls, is this correct?

Reply:
Updated now @MatthewAlexanderBeck  :+1:

Reply:
Cheers